### PR TITLE
second_branch Init second.py

### PR DESCRIPTION
**Pull request by lawrencebusiness1997**

### CODE REVIEW

**Comment by lawrencelee0205** [comment]
Please change the second.py

File: second.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("If you see this, then you are in second branch")
```
Comment (lawrencelee0205) [inline second.py:1]: You should add a line to print "Hello World"

Comment (lawrencebusiness1997) [reply second.py:1]: I agree

**Comment by lawrencelee0205** [comment]
LGTM

**Pull request merged**